frontend/src/app/testQueue/testCardTestConstants.ts

```diff
@@ -83,7 +83,7 @@ const GENDER_SEXUAL_PARTNERS_FEMALE_OVERRIDE = {
 };
 const BLURRED_VISION_OVERRIDE = {
   symptoms:
-    '{"15188001":false,"26284000":false,"46636008":true,"56940005":false,"68225006":false,"91554004":false,"195469007":false,"266128007":false,"724386005":false}',
+    '{"15188001":false,"26284000":false,"56317004":false,"56940005":false,"91554004":false,"195469007":false,"246636008":true,"266128007":false,"724386005":false}',
```

Comment:
⚠️ the change here is `Alopecia (SNOMED: 56317004)`, the snomed for blurred vision is corrected from `46636008` to `246636008` and the order of the others were changed